integrations/airflow/README.md

```diff
@@ -42,40 +42,27 @@ $ python3 setup.py install
 
 ## Configuration
 
-The library depends on a _backend_. A `Backend` is configurable and lets the library know where to write dataset, job, and run metadata.
 
-### Backends
-
-* `HTTP`: Write metadata to Marquez
-* `FILE`: Write metadata to a file (as `json`) under `/tmp/marquez`
-* `LOG`: Simply just logs the metadata to the console
-
-By default, the `HTTP` backend will be used (see next sections on configuration). To override the default backend and write metadata to a file, use `MARQUEZ_BACKEND`:
+### `HTTP` Backend Environment Variables
 
-```
-MARQUEZ_BACKEND=FILE
-```
+`marquez-airflow` uses OpenLineage client to push data to Marquez.
 
-> **Note:** Metadata will be written to `/tmp/marquez/client.requests.log`, but the location can be overridden with `MARQUEZ_FILE`.
+OpenLineage client depends on environment variables:
 
-### `HTTP` Backend Authentication
+* `OPENLINEAGE_URL` - point to service which will consume OpenLineage events
+* `OPENLINEAGE_PRODUCER` - name of producer that client will send along with your events
```

Comment:
We'll be dropping support for `OPENLINEAGE_PRODUCER`, see https://github.com/OpenLineage/OpenLineage/issues/72. Mainly, it's to ensure callers configure the producer via the `py` client constructor. Though, `OPENLINEAGE_URL` and `OPENLINEAGE_API_KEY` make sense to keep as env vars, but also have support to set via the client constructor.

Reply:
Added notification. Although, `marquez_airflow` users don't configure connection by client themselves now. This would require changes of how users use Airflow integration - aka it won't be just drop-in `from marquez_airflow import DAG`.

On the other hand, this is good - we have to change it anyway to support 2.0 and stuff like [taskflow API](https://airflow.apache.org/docs/apache-airflow/stable/tutorial_taskflow_api.html)